app/views/spree/admin/orders/_list.html.haml

```diff
@@ -0,0 +1,16 @@
+%table.payment-list
+  %thead
+    %tr
+      %th= "#{Spree.t('date')}/#{Spree.t('time')}"
```

Comment:
always use lazylookups for translations and never Spree.t (it adds the spree namespace), always use I18n or in this case, just t
this would be t('.date') and t('.time') or maybe just t('.datetime') as it is specific to this template.
and then you can add the key to en.yml in spree.admin.order.payments_list.date around line 3236 in en.yml

Reply:
more info here: https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n)#development

Reply:
Cool - thanks for the link to the specific wiki docs, that helps a lot. I updated the docs a bit to make them more clear

Reply:
:+1: nice, thanks for updating the docs :ok_hand: 
There was a dot missing in the example, lazy lookups need the initial dot:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n)/_compare/61e41777a1596cd032bcc05fb1f5e7c1786f6f2f